validationFunctions/validateCharityNumber.js

```diff
@@ -23,8 +23,7 @@ const validateCharityNumber = charityNumber => {
       return false;
     }
     if (
-      noWhiteSpaceCharityNumber.length > 8 ||
-      noWhiteSpaceCharityNumber.length < 6
+      noWhiteSpaceCharityNumber.length !== 8
```

Comment:
Is this change correct?  It looks like valid UK charity numbers can still be between 6 and 8 digits...

Reply:
Apparently a lot of them are left padded with 0s to make 8 characters 
https://twitter.com/kanedr/status/1229707338132729856